eventarc/audit-storage/src/main/java/com/example/cloudrun/EventController.java

```diff
@@ -37,7 +36,7 @@
 
   @RequestMapping(value = "/", method = RequestMethod.POST)
   public ResponseEntity<String> receiveMessage(
-      @RequestBody PubSubBody body, @RequestHeader Map<String, String> headers) {
+      @RequestBody Map<String, String> body, @RequestHeader Map<String, String> headers) {
```

Comment:
I forgot the fields aren't necessary strings, so this should be <String, Object> for the unmarshalling. Will approve for this simple change.

Reply:
Changed to `Object`.